source/includes/connection-snippets/go-connection.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
```

Comment:
nit: Imports should be organized into two sections separated by a blank line. Each section should be in alphabetical order:

```
"context"
"fmt"
"log"
"time"

"go.mongodb.org/mongo-driver/mongo"
...
```

Reply:
Ah ok, thanks, I incorrectly thought `gofmt` would take care of ordering.